`build_binutils.sh` fails to build GDB because `PySys_SetPath` was removed in Python 3.13

If Python 3.13+ is installed, `build_binutils.sh` fails with
```
In file included from ../../gdb/python/py-breakpoint.c:22:
../../gdb/python/python-internal.h: In function 'void gdb_PySys_SetPath(const wchar_t*)':
../../gdb/python/python-internal.h:279:3: error: 'PySys_SetPath' was not declared in this scope
  279 |   PySys_SetPath (const_cast<GDB_PYSYS_SETPATH_CHAR *> (path));
      |   ^~~~~~~~~~~~~
```

GDB was fixed in July 2022 with [fe587fc997af5486e94250a3dbe7e96f966b1d94](https://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git;h=fe587fc997af5486e94250a3dbe7e96f966b1d94). Can GDB be updated to a newer version? (osxcross uses 8.3, latest is 16.2)

--- a/build_binutils.sh
+++ b/build_binutils.sh
@@ -22,7 +22,7 @@ fi
 
 # gdb version to build
 if [ -z "$GDB_VERSION" ]; then
-  GDB_VERSION=8.3
+  GDB_VERSION=16.2
 fi
 
 # architecture to target

